libs/wingsdk/src/util/util.ts

```diff
@@ -239,10 +238,10 @@ export class Util {
     };
 
     try {
-      const { stdout, stderr } = await execFilePromise(program, args, execOpts);
+      const stdout = execFileSync(program, args, execOpts);
       return {
         stdout: stdout.toString(),
-        stderr: stderr.toString(),
+        stderr: "",
```

Comment:
Maybe add a TODO comment or open an issue to note this

Reply:
There is no real limitation I think- the `stderr` is thrown in the catch statement if exists- and passed to the user :) 
I think that it yields the same result of the `promisify` function

Reply:
@tsuf239 I thought even if a command succeeds it can still emit to both stdout and stderr?